Clean/Gadgets/Rotation32/Theorems.lean

```diff
@@ -19,15 +19,15 @@ each resulting (high, low) pair again.
 The ultimate goal is to prove that this is equivalent to `rotRight32`.
 -/
 def rotRight32_bytes (xs : Vector ℕ 4) (o : ℕ) : Vector ℕ 4 :=
-  .ofFn fun ⟨ i, hi ⟩ => xs[i] / 2^o + (xs[(i + 1) % 4] % 2^o) * 2^(8-o)
+  .ofFn fun ⟨ i, hi ⟩ => xs[i] / 2^o + (xs[(i + 1) % 4] % 2^o) * 2^(8 - o)
 
 -- unfold what rotRight32_bytes does on a U32
 def rotRight32_u32 : U32 ℕ → ℕ → U32 ℕ
   | ⟨ x0, x1, x2, x3 ⟩, o => ⟨
-    (x0 / 2^o) + (x1 % 2^o) * 2^(8-o),
-    (x1 / 2^o) + (x2 % 2^o) * 2^(8-o),
-    (x2 / 2^o) + (x3 % 2^o) * 2^(8-o),
-    (x3 / 2^o) + (x0 % 2^o) * 2^(8-o),
+    (x0 / 2^o) + (x1 % 2^o) * 2^(8 - o),
+    (x1 / 2^o) + (x2 % 2^o) * 2^(8 - o),
+    (x2 / 2^o) + (x3 % 2^o) * 2^(8 - o),
+    (x3 / 2^o) + (x0 % 2^o) * 2^(8 - o),
```

Comment:
I remember personally removing these spaces because I felt being in an exponent (which definitely should be written without spaces) meant the inner operations should look more "compressed" as well

Reply:
ohhh, makes sense.

Reply:
but I'm open to changing it for consistency with other cases of `+` and `-`